src/utils/api.ts

```diff
@@ -25,6 +25,11 @@ export const getErrorMessage = (e: any) => {
     message = get(e, 'message')
   }
 
+  if (!message && Array.isArray(e)) {
+    const error = e.find(err => err.status >= 400)
+    message = get(error, 'data.message')
+  }
```

Comment:
Added this case for when an array of promises are returned and we need to identify the error message from one of the promises